tox.ini

```diff
@@ -37,9 +37,9 @@ source =
 
 [flake8]
 max-line-length = 90
-extend-ignore = S101, D101, D102, D103, D105, D106, D107, TC006
+extend-ignore = S101, D101, D102, D103, D105, D106, D107
```

Comment:
finally realized why they want to stringify types in `cast` - the types aren't going to be imported at runtime. And is probably a minor speedup as well